### PR TITLE
[Security] Added note inside phpdoc.

### PR DESCRIPTION
**Pull request by Koc**
| Q | A |
| --- | --- |
| Branch? | 2.8 |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Authorization/Voter/Voter.php [hunk 1]
```diff
@@ -74,6 +74,7 @@ public function vote(TokenInterface $token, $object, array $attributes)
 
     /**
      * Perform a single access check operation on a given attribute, subject and token.
+     * It is safe to assume that $attribute and $subject pass supports method call.
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Security/Core/Authorization/Voter/Voter.php]: I don't understand this sentence, sorry :) Does anyone have a better proposal?

Comment (Koc) [inline src/Symfony/Component/Security/Core/Authorization/Voter/Voter.php]: `supports` method calling before `voteOnAttribute` and if `supports` returns `false` then `voteOnAttribute` would not be executed. And there is no sense duplicate checks inside `voteOnAttribute`.

Comment (nicolas-grekas) [inline src/Symfony/Component/Security/Core/Authorization/Voter/Voter.php]: Oh :) What about:
It is safe to assume that $attribute and $subject already passed the "supports()" method check.

Comment (fabpot) [inline src/Symfony/Component/Security/Core/Authorization/Voter/Voter.php]: 👍

Comment (Koc) [inline src/Symfony/Component/Security/Core/Authorization/Voter/Voter.php]: Updated

**Comment by fabpot** [comment]
Isn't it something for 2.7?

**Comment by nicolas-grekas** [comment]
The Voter class doesn't exist in 2.7

**Comment by fabpot** [comment]
Thank you @Koc.

**Pull request closed without merging**